client/src/pages/Permits/index.js

```diff
@@ -4,16 +4,22 @@ import Wrapper from 'components/Wrapper';
 import React from 'react';
 import { createColumnHelper } from '@tanstack/react-table'
 import { useTranslation } from 'locale/he_IL';
-import { fetchPermits } from './controller';
+import { CheckIfUserCanAccessPage } from 'hooks';
+import api from 'services/api';
 
 const Homepage = () => {
+	CheckIfUserCanAccessPage();
+
 	const { t } = useTranslation();
 	const [ data, setData ] = useState([]);
 
 	useEffect(() => {
-		const fetchData = async () => {
-			const res = await fetchPermits();
-			setData(res.data);
+		const fetchData = () => {
+			return api.get('/permit')
+				.then(result => {
+					setData(result.data);
+				})
+				.catch(error => console.error(error));
```

Comment:
Please remove console error, also, consider adding the useData hook
```
https://medium.com/swlh/creating-react-hook-for-fetching-data-4193d8479138
```

Reply:
What should be there instead of the `console.error`? a `console.log`? I went according to what you have in different parts of the code:
client/src/pages/UserPlans/index.jsx line 24 
or 
client/src/scenes/alerts/Alerts.jsx line 45

Regarding adding a `useData` hook - I'm not sure exactly how you want to set the loading animation, or what a returned error would look for Meirim. I'll be happy to sit with you about this.

Reply:
just remove logs for now